### DISCUSSION TITLE
AxisItem wheel not working after resize

### DISCUSSION THREAD

**Discussion by danielhrisca** [pyqtgraph/pyqtgraph#2280] [Q&A]
Hello,

I use this derived class https://github.com/danielhrisca/asammdf/blob/8da36f56aebcc2b0cd9d23f82edfeee3a1d7ec3d/asammdf/gui/widgets/formated_axis.py#L16 were I've added buttons to change the vertical axis width

When I increase the width the wheel events are only registered in the initial AxisItem rect. 

![wheel](https://user-images.githubusercontent.com/20952040/165953167-0c16a48e-7ab9-4986-b658-96286d5bde86.gif)

Any ideas what might be the cause?

**Comment by danielhrisca** [comment]
For some reason the wheel event gets handled by the PlotItem instead of the AxisItem. I had to monkey path the wheelEvent of the PlotItem to forward the event to the AxisItem.

**Reply by j9ac9k** [reply]
Hi @danielhrisca we recently made some changes you can see a fantastically confusing discussion on #2241 but AFAIK `master` branch should be behaving "correctly".

**Reply by danielhrisca** [reply]
I've just tried the latest ``master`` code and it still does not work for me

**Reply by j9ac9k** [reply]
hmm... if you can provide a reproducible example, I can attempt to reproduce and diagnose.  I suspect that PR #2057 is what is impacting your use case, and I _suspect_ there is a "better" way to address your issue, but w/o an example I'm going to just be making educated guesses.
